tests/flow/new_react/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -865,11 +865,7 @@ var ReactClass = React.createClass({
   render: function(): any {
     // Any state access here seems to make state any
     this.state;
-    return (
-      <div>
-        {this.state.bar.qux}
-      </div>
-    );
+    return <div>{this.state.bar.qux}</div>;
```

Comment:
I'll test this out on some open source projects and see which approach gives less changes. Although that doesn't have to be the deciding factor!